jobs/JGDAS_ENKF_ECEN

```diff
@@ -80,7 +80,7 @@ fi
 export OPREFIX="${CDUMP}.t${cyc}z."
 export APREFIX="${CDUMP}.t${cyc}z."
 export APREFIX_ENKF="${CDUMP_ENKF}.t${cyc}z."
-export GPREFIX="${CDUMP}.t${gcyc}z."
```

Comment:
Will this not mess up late-cycle?

Reply:
I am not sure, however, I think we can add some if statements to make it better for both early and late enkf

Reply:
This change still allows both early and late cycle to point to previous gdas directory correctly

Reply:
Done. Thanks @WalterKolczynski-NOAA @TravisElless-NOAA